### PR TITLE
Country select / update tsconfig

### PR DESCRIPTION
**Pull request by minotogna**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -39,8 +39,9 @@
         "tsx": "never"
       }
     ],
-    "no-unused-vars": ["error", { "argsIgnorePattern": "^_$" }],
+    "no-unused-vars": "off",
```
Comment (sorja) [inline .eslintrc.json:42]: Why do we disable this check?

File: common/countryRole.ts [hunk 2]
```diff
@@ -32,7 +32,7 @@ export const roles = {
 
 export const roleKeys = R.pipe(R.values, R.map(R.prop('role')))(roles)
 
-export const getRoleLabelKey = (roleName: any) => R.path([roleName, 'labelKey'], roles)
+export const getRoleLabelKey = (roleName: any): string => R.path([roleName, 'labelKey'], roles)
```
Comment (sorja) [inline common/countryRole.ts:35]: Should we remove ramda and use ex. roles?.[roleName]?.labelKey

Comment (minotogna) [reply common/countryRole.ts:35]: future refactor yes

File: server/repository/odp/readEofOdps.ts [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import * as db from '@server/db/db_deprecated'
 import * as R from 'ramda'
 
-export const eofReducer = (results: any, row: any, type = 'fra') => [
+export const eofReducer = (results: any, row: any, _type = 'fra') => [
```
Comment (sorja) [inline server/repository/odp/readEofOdps.ts:4]: last unused param can be removed?

File: server/repository/odp/readFocOdps.ts [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import * as db from '@server/db/db_deprecated'
 import * as R from 'ramda'
 
-export const focReducer = (results: any, row: any, type = 'fra') => [
+export const focReducer = (results: any, row: any, _type = 'fra') => [
```
Comment (sorja) [inline server/repository/odp/readFocOdps.ts:4]: last unused param can be removed?

File: server/repository/review/reviewRepository.ts [hunk 5]
```diff
@@ -160,7 +160,7 @@ export const createIssueWithComment = (
   `,
         [countryIso, section, target, 'opened']
       )
-      .then((res: any) => client.query(`SELECT last_value FROM issue_id_seq`))
+      .then((_res: any) => client.query(`SELECT last_value FROM issue_id_seq`))
```
Comment (sorja) [inline server/repository/review/reviewRepository.ts:163]: last unused param can be removed?

File: server/repository/odp/readEofOdps.ts (hunk 3 above)
Comment (minotogna) [reply server/repository/odp/readEofOdps.ts:4]: future refactor

File: webapp/pages/Landing/Introduction/AreaSelector/AreaSelector.tsx [hunk 6]
```diff
@@ -33,32 +35,42 @@ const AreaSelector: React.FC = () => {
...
+          <DropdownAreas
+            area={areas.countries}
+            areaISOs={countryISOs}
+            assessmentType={FRA.type}
+            dropdownOpened={dropdownOpened}
+            setDropdownOpened={setDropdownOpened}
+          />
+        ) : (
+          <SelectMobile area={areas.countries} areaISOs={countryISOs} assessmentType={FRA.type} />
+        )}
```
Comment (sorja) [inline webapp/pages/Landing/Introduction/AreaSelector/AreaSelector.tsx:73]: Explore ResponsiveComponent (idea) 💡

**Review by sorja — APPROVED** [review]

File: webapp/pages/Landing/Introduction/AreaSelector/AreaSelector.tsx (hunk 6 above)
Comment (minotogna) [reply webapp/pages/Landing/Introduction/AreaSelector/AreaSelector.tsx:73]: sure, feel free to make a proposal

**Pull request merged**